src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TANInputViewController/TanInputViewModel.swift

```diff
@@ -116,5 +114,22 @@ final class TanInputViewModel {
 		default: return nil
 		}
 	}
-
+	
+	private func registerPCRTest() {
+		isPrimaryButtonEnabled = false
+		isPrimaryBarButtonIsLoading = true
+		coronaTestService.registerPCRTest(teleTAN: text, isSubmissionConsentGiven: false) { [weak self] result in
+			switch result {
+			case let .failure(error):
+				// If teleTAN is incorrect, show Alert Controller
```

Comment:
is in one of the following methods (like the alert) some error logged? If not, perhaps add some logs here...

Reply:
Done